controllers/nodefeaturediscovery_controls.go

```diff
@@ -109,9 +109,8 @@ func ServiceAccount(n NFD) (ResourceStatus, error) {
 
 	// found states if the ServiceAccount was found
 	found := &corev1.ServiceAccount{}
-	logger := log.WithValues("ServiceAccount", obj.Name, "Namespace", obj.Namespace)
 
-	logger.Info("Looking for")
+	klog.Info("Looking for ServiceAccount %q on Namespace %q", obj.Name, obj.Namespace)
```

Comment:
supert nit: in/on namespace, use consistently 😄 

```suggestion
	klog.Info("Looking for ServiceAccount %q in Namespace %q", obj.Name, obj.Namespace)
```